src/process/LinuxLokinetProcessManager.cpp

```diff
@@ -3,7 +3,7 @@
 #include <QProcess>
 #include <QDebug>
 
-#ifdef Q_OS_LINUX
+#if !defined(Q_OS_WIN) && !defined(Q_OS_MACOS)
 
```

Comment:
Are there other platforms that you tested this on where this works?

Reply:
netbsd (openbsd), freebsd